apps/browser/components/Sidebar.tsx

```diff
@@ -76,10 +75,12 @@ const Sidebar = ({
             </button>
           )}
         </Upload>
+
+        <ChatList />
```

Comment:
I think this shouldn't be rendered if the user only has entered an OpenAI key and doesn't have a user ID. If not, they could click "new chat" and experience errors.

Reply:
agreed, updated